Analysis/Tasks/PWGCF/o2femtodream/include/o2femtodream/FemtoDerived.h

```diff
@@ -0,0 +1,84 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef ANALYSIS_TASKS_PWGCF_FEMTODREAM_FEMTODERIVED_H_
+#define ANALYSIS_TASKS_PWGCF_FEMTODREAM_FEMTODERIVED_H_
+
+#include "Framework/ASoA.h"
+#include "AnalysisDataModel/Multiplicity.h"
+#include "Framework/AnalysisDataModel.h"
+#include "AnalysisDataModel/TrackSelectionTables.h"
+#include "AnalysisDataModel/PID/PIDResponse.h"
+
+namespace o2::aod
+{
+
+namespace femtodreamcollision
+{
+DECLARE_SOA_COLUMN(MultV0M, multV0M, float);
+DECLARE_SOA_COLUMN(Sphericity, sphericity, float);
+} // namespace femtodreamcollision
+DECLARE_SOA_TABLE(FemtoDreamCollisions, "AOD", "FemtoDreamCols",
+                  o2::soa::Index<>,
+                  o2::aod::collision::PosZ,
+                  femtodreamcollision::MultV0M,
+                  femtodreamcollision::Sphericity);
+using FemtoDreamCollision = FemtoDreamCollisions::iterator;
+
+namespace femtodreamparticle
+{
+DECLARE_SOA_INDEX_COLUMN(FemtoDreamCollision, femtoDreamCollision);
+DECLARE_SOA_COLUMN(pT, pt, float);
+DECLARE_SOA_COLUMN(Eta, eta, float);
+DECLARE_SOA_COLUMN(Phi, phi, float);
+DECLARE_SOA_COLUMN(Cut, cut, unsigned long int);
+DECLARE_SOA_COLUMN(TempFitVar, tempFitVar, float);
+// debug variables
+DECLARE_SOA_COLUMN(Sign, sign, int);
```

Comment:
int8 should be enough here

Reply:
After all these columns here are just for debugging, i.e. we certainly won't write out these for a large fraction of the data. We optimise anyways?

Reply:
If the columns are "optional" it would make sense to have them in a separate table, joinable to the main one. And using types with the smallest required size is indeed important due to memory restrictions when running in the Grid.

Reply:
That was exactly the idea. In any case, we're now using the smaller data types